public/locales/en/actionpanel.json

```diff
@@ -0,0 +1,5 @@
+{
+  "breadcrumbs-aria-label": "Back",
```

Comment:
should we maybe also add the title of each planet to the top of each respective json file?  Now that one is not being translated